### PR TITLE
Add separate certificates section #433

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
Adding section on separate certificates for client and server on transport TLS layer

### Issues Resolved
Part of issue here https://github.com/opensearch-project/documentation-website/issues/433

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
@scrawfor99 - Can you please review? @AntonEliatra  - please ignore the Vale errors for now. Thanks!

File: _security/configuration/tls.md [hunk 1]
```diff
@@ -77,6 +76,43 @@ Name | Description
 `plugins.security.ssl.http.truststore_password` | Truststore password. Default is `changeit`.
 
 
+## Separate client and server certificates for transport layer TLS
+
+By default, transport layer TLS certificates need to be configured as both client (`TLS Web Client Authentication`) and server (`TLS Web Server Authentication`) in the certificate's `Extended Key Usage` section, as they take responsibility as server and client in internal communication between nodes.
+If you want to use separate certificates as client and server, you need to add following line to `opensearch.yml` and settings outlined in [separate client and server X.509 PEM certificates and PKCS #8 keys]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-x509-pem-certificates-and-pkcs-8-keys) or [separate client and server Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-keystore-and-truststore-files)
```
Comment (scrawfor99) [inline _security/configuration/tls.md:82]:
```suggestion
If you want to use separate certificates as client and server, you need to add following line to `opensearch.yml` and settings outlined in [separate client and server X.509 PEM certificates and PKCS #8 keys]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-x509-pem-certificates-and-pkcs-8-keys) or [separate client and server Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-keystore-and-truststore-files)
```
```suggestion
If you want to use separate client and server certificates, you need to add following line and settings to `opensearch.yml`. You can also find steps on creating [separate client and server X.509 PEM certificates and PKCS #8 keys]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-x509-pem-certificates-and-pkcs-8-keys) or [separate client and server Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-keystore-and-truststore-files) by following the hotlinks.
```

**Review by scrawfor99 — APPROVED** [review]
Looks good from the technical details perspective. The two intro lines seem a little confusing to me so maybe reword those.

File: _security/configuration/tls.md (hunk 1 above)
Comment (AntonEliatra) [reply _security/configuration/tls.md]: @scrawfor99 Thanks for the above, this part is a bit tricky, as main opensearch.yml line is common in both cases, and the two options that follow are different.

Do you think below is more clear

`If you want to use separate certificates as client and server, you need to add `plugins.security.ssl.transport.extended_key_usage_enabled: true` to `opensearch.yml`, then configure settings outlined in [separate client and server X.509 PEM certificates and PKCS #8 keys]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-x509-pem-certificates-and-pkcs-8-keys) or [separate client and server Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-keystore-and-truststore-files)`

File: _security/configuration/tls.md [hunk 2]
```diff
@@ -77,6 +76,41 @@ Name | Description
 `plugins.security.ssl.http.truststore_password` | Truststore password. Default is `changeit`.
 
 
+## Separate client and server certificates for transport layer TLS
+
+By default, transport layer TLS certificates need to be configured as both client (`TLS Web Client Authentication`) and server (`TLS Web Server Authentication`) in the certificate's `Extended Key Usage` section, as they take responsibility as server and client in internal communication between nodes.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:81]:
```suggestion
By default, transport layer TLS certificates need to be configured as both client (`TLS Web Client Authentication`) and server (`TLS Web Server Authentication`) in the certificate's `Extended Key Usage` section, because the certificates are responsible for the server's and client's internal communication between nodes.
```

File: _security/configuration/tls.md [hunk 3]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+If you want to use separate certificates as client and server, you need to add `plugins.security.ssl.transport.extended_key_usage_enabled: true` to `opensearch.yml`, then configure settings outlined in [separate client and server X.509 PEM certificates and PKCS #8 keys]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-x509-pem-certificates-and-pkcs-8-keys) or [separate client and server Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-keystore-and-truststore-files)
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:82]:
```suggestion
If you want to use separate certificates as client and server,  add  the `plugins.security.ssl.transport.extended_key_usage_enabled: true` setting to `opensearch.yml`. Next, configure the settings outlined in the [separate client and server X.509 PEM certificates and PKCS #8 keys]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-x509-pem-certificates-and-pkcs-8-keys) or [separate client and server Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-keystore-and-truststore-files) sections.
```

File: _security/configuration/tls.md [hunk 4]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+
+### Separate client and server X.509 PEM certificates and PKCS #8 keys
+
+Name | Description
+:--- | :---
+`plugins.security.ssl.transport.server.pemkey_filepath` | Path to the server certificate's key file (PKCS \#8), which must be under the `config` directory, specified using a relative path. Required.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:88]:
```suggestion
`plugins.security.ssl.transport.server.pemkey_filepath` | Path to the server certificate's key file (PKCS \#8). Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 5]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.server.pemkey_password` | Server key password. Omit this setting if the key has no password. Optional.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:89]:
```suggestion
`plugins.security.ssl.transport.server.pemkey_password` | The server key password. Omit this setting if the key has no password. Optional.
```

File: _security/configuration/tls.md [hunk 6]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.server.pemcert_filepath` | Path to the X.509 node server certificate chain (PEM format), which must be under the `config` directory, specified using a relative path. Required.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:90]:
```suggestion
`plugins.security.ssl.transport.server.pemcert_filepath` | Path to the X.509 node server certificate chain (PEM format). Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 7]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.server.pemtrustedcas_filepath` | Path to the root CAs (PEM format), which must be under the `config` directory, specified using a relative path. Required.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:91]:
```suggestion
`plugins.security.ssl.transport.server.pemtrustedcas_filepath` | Path to the root CAs (PEM format). Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 8]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.client.pemkey_filepath` | Path to the client certificate's key file (PKCS \#8), which must be under the `config` directory, specified using a relative path. Required.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:92]:
```suggestion
`plugins.security.ssl.transport.client.pemkey_filepath` | Path to the client certificate's key file (PKCS \#8). Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 9]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.client.pemkey_password` | Client key password. Omit this setting if the key has no password. Optional.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:93]:
```suggestion
`plugins.security.ssl.transport.client.pemkey_password` | The client key password. Omit this setting if the key has no password. Optional.
```

File: _security/configuration/tls.md [hunk 10]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.client.pemcert_filepath` | Path to the X.509 node client certificate chain (PEM format), which must be under the `config` directory, specified using a relative path. Required.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:94]:
```suggestion
`plugins.security.ssl.transport.client.pemcert_filepath` | Path to the X.509 node client certificate chain (PEM format). Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 11]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.client.pemtrustedcas_filepath` | Path to the root CAs (PEM format), which must be under the `config` directory, specified using a relative path. Required.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:95]:
```suggestion
`plugins.security.ssl.transport.client.pemtrustedcas_filepath` | Path to the root CAs (PEM format). Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 12]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+
+### Separate client and server keystore and truststore files
+
+Name | Description
+:--- | :---
+`plugins.security.ssl.transport.keystore_type` | The type of the keystore file, JKS or PKCS12/PFX. Optional. Default is JKS.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:101]:
```suggestion
`plugins.security.ssl.transport.keystore_type` | The type of the keystore file, either `JKS` or `PKCS12/PFX`. Optional. Default is `JKS`.
```

File: _security/configuration/tls.md [hunk 13]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.keystore_filepath` | Path to the keystore file, which must be under the `config` directory, specified using a relative path. Required.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:102]:
```suggestion
`plugins.security.ssl.transport.keystore_filepath` | Path to the keystore file. Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 14]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.server.keystore_alias` | Alias name for server key. Optional. Default is the first alias.
+`plugins.security.ssl.transport.client.keystore_alias` | Alias name for client key. Optional. Default is the first alias.
+`plugins.security.ssl.transport.server.keystore_keypassword` | Keystore password for server. Default is `changeit`.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:105]:
```suggestion
`plugins.security.ssl.transport.server.keystore_keypassword` | The keystore password for the server. Default is `changeit`.
```

File: _security/configuration/tls.md [hunk 15]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.client.keystore_keypassword` | Keystore password for client. Default is `changeit`.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:106]:
```suggestion
`plugins.security.ssl.transport.client.keystore_keypassword` | The keystore password for the client. Default is `changeit`.
```

File: _security/configuration/tls.md [hunk 16]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.client.pemcert_filepath` | Path to the X.509 node client certificate chain (PEM format), which must be under the `config` directory, specified using a relative path. Required.
+`plugins.security.ssl.transport.client.pemtrustedcas_filepath` | Path to the root CAs (PEM format), which must be under the `config` directory, specified using a relative path. Required.
+
+### Separate client and server keystore and truststore files
+
+Name | Description
+:--- | :---
+`plugins.security.ssl.transport.keystore_type` | The type of the keystore file, JKS or PKCS12/PFX. Optional. Default is JKS.
+`plugins.security.ssl.transport.keystore_filepath` | Path to the keystore file, which must be under the `config` directory, specified using a relative path. Required.
+`plugins.security.ssl.transport.server.keystore_alias` | Alias name for server key. Optional. Default is the first alias.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:103]:
```suggestion
`plugins.security.ssl.transport.server.keystore_alias` | The alias name of the server key. Optional. Default is the first alias.
```

File: _security/configuration/tls.md [hunk 17]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.client.keystore_alias` | Alias name for client key. Optional. Default is the first alias.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:104]:
```suggestion
`plugins.security.ssl.transport.client.keystore_alias` | The alias name for the client key. Optional. Default is the first alias.
```

File: _security/configuration/tls.md [hunk 18]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.server.keystore_keypassword` | Keystore password for server. Default is `changeit`.
+`plugins.security.ssl.transport.client.keystore_keypassword` | Keystore password for client. Default is `changeit`.
+`plugins.security.ssl.transport.server.truststore_alias` | Alias name for server. Optional. Default is all certificates.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:107]:
```suggestion
`plugins.security.ssl.transport.server.truststore_alias` | The alias name of the server. Optional. Default is all certificates.
```

File: _security/configuration/tls.md [hunk 19]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.client.truststore_alias` | Alias name for client. Optional. Default is all certificates.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:108]:
```suggestion
`plugins.security.ssl.transport.client.truststore_alias` | The alias name of the client. Optional. Default is all certificates.
```

File: _security/configuration/tls.md [hunk 20]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.truststore_filepath` | Path to the truststore file, which must be under the `config` directory, specified using a relative path. Required.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:109]:
```suggestion
`plugins.security.ssl.transport.truststore_filepath` | The path to the `truststore` file. Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 21]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.truststore_type` | The type of the truststore file, JKS or PKCS12/PFX. Default is JKS.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:110]:
```suggestion
`plugins.security.ssl.transport.truststore_type` | The type of the `truststore` file, either `JKS` or `PKCS12/PFX`. Default is `JKS`.
```

File: _security/configuration/tls.md [hunk 22]
```diff
@@ -77,6 +76,41 @@ Name | Description
...
+`plugins.security.ssl.transport.truststore_password` | Truststore password. Default is `changeit`.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:111]:
```suggestion
`plugins.security.ssl.transport.truststore_password` | The `truststore` password. Default is `changeit`.
```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]
We could make similar changes to the other sections on the page. That's up to you though.

File: _security/configuration/tls.md (hunk 2 above)
Comment (AntonEliatra) [reply _security/configuration/tls.md:81]: @Naarcha-AWS would it be a better explanation to say instead of:

```
-because the certificates are responsible for the server's and client's internal communication between nodes.

+bacause the nodes using the TLS certificates take on the responsibility of serving and receiving the communication requests internally.
```

**Review by Naarcha-AWS — APPROVED** [review]

File: _security/configuration/tls.md [hunk 23]
```diff
@@ -54,27 +54,61 @@ Name | Description
...
+`plugins.security.ssl.http.truststore_password` | The password for the truststore. Default is `changeit`.
+
+
+## Separate client and server certificates for transport layer TLS
+
+By default, transport layer TLS certificates need to be configured as both client (`TLS Web Client Authentication`) and server (`TLS Web Server Authentication`) in the certificate's `Extended Key Usage` section, bacause the nodes using the TLS certificates take on the responsibility of serving and receiving the communication requests internally.
+If you want to use separate certificates as client and server,  add  the `plugins.security.ssl.transport.extended_key_usage_enabled: true` setting to `opensearch.yml`. Next, configure the settings outlined in the [separate client and server X.509 PEM certificates and PKCS #8 keys]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-x509-pem-certificates-and-pkcs-8-keys) or [separate client and server Keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-keystore-and-truststore-files) sections.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:82]:
```suggestion
If you want to use separate certificates as client and server,  add the `plugins.security.ssl.transport.extended_key_usage_enabled: true` setting to `opensearch.yml`. Next, configure the settings outlined in the [separate client and server X.509 PEM certificates and PKCS #8 keys]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-x509-pem-certificates-and-pkcs-8-keys) or [separate client and server keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-keystore-and-truststore-files) sections.
```

File: _security/configuration/tls.md [hunk 24]
```diff
@@ -54,27 +54,61 @@ Name | Description
...
 `plugins.security.ssl.http.truststore_filepath` | Path to the truststore file, which must be under the `config` directory, specified using a relative path. Required.
-`plugins.security.ssl.http.truststore_alias` | Alias name. Optional. Default is all certificates.
-`plugins.security.ssl.http.truststore_password` | Truststore password. Default is `changeit`.
+`plugins.security.ssl.http.truststore_alias` | The alias name of the truststore. Optional. Default is all certificates.
+`plugins.security.ssl.http.truststore_password` | The password for the truststore. Default is `changeit`.
+
+
+## Separate client and server certificates for transport layer TLS
+
+By default, transport layer TLS certificates need to be configured as both client (`TLS Web Client Authentication`) and server (`TLS Web Server Authentication`) in the certificate's `Extended Key Usage` section, bacause the nodes using the TLS certificates take on the responsibility of serving and receiving the communication requests internally.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:81]:
```suggestion
By default, transport layer TLS certificates need to be configured as both client (`TLS Web Client Authentication`) and server (`TLS Web Server Authentication`) in the certificate's `Extended Key Usage` section, because the nodes using the TLS certificates take on the responsibility of serving and receiving the communication requests internally.
```

File: _security/configuration/tls.md [hunk 25]
```diff
@@ -54,27 +54,61 @@ Name | Description
...
+If you want to use separate certificates as client and server,  add the `plugins.security.ssl.transport.extended_key_usage_enabled: true` setting to `opensearch.yml`. Next, configure the settings outlined in the [separate client and server X.509 PEM certificates and PKCS #8 keys]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-x509-pem-certificates-and-pkcs-8-keys) or [separate client and server keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-keystore-and-truststore-files) sections.
```
Comment (Naarcha-AWS) [inline _security/configuration/tls.md:82]:
```suggestion
If you want to use separate certificates for client and server, add the `plugins.security.ssl.transport.extended_key_usage_enabled: true` setting to `opensearch.yml`. Next, configure the settings outlined in the [separate client and server X.509 PEM certificates and PKCS #8 keys]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-x509-pem-certificates-and-pkcs-8-keys) or [separate client and server keystore and truststore files]({{site.url}}{{site.baseurl}}/security/configuration/tls/#separate-client-and-server-keystore-and-truststore-files) sections.
```

File: _security/configuration/tls.md [hunk 26]
```diff
@@ -25,7 +25,7 @@ The following tables contain the settings you can use to configure the location
 Name | Description
 :--- | :---
 `plugins.security.ssl.transport.pemkey_filepath` | Path to the certificate's key file (PKCS \#8), which must be under the `config` directory, specified using a relative path. Required.
-`plugins.security.ssl.transport.pemkey_password` | Key password. Omit this setting if the key has no password. Optional.
+`plugins.security.ssl.transport.pemkey_password` | The key password. Omit this setting if the key has no password. Optional.
 `plugins.security.ssl.transport.pemcert_filepath` | Path to the X.509 node certificate chain (PEM format), which must be under the `config` directory, specified using a relative path. Required.
 `plugins.security.ssl.transport.pemtrustedcas_filepath` | Path to the root CAs (PEM format), which must be under the `config` directory, specified using a relative path. Required.
```
Comment (natebower) [inline _security/configuration/tls.md:30]:
```suggestion
`plugins.security.ssl.transport.pemtrustedcas_filepath` | Path to the root certificate authorities (CAs) (PEM format), which must be under the `config` directory, specified using a relative path. Required.
```

File: _security/configuration/tls.md [hunk 27]
```diff
@@ -54,27 +54,61 @@ Name | Description
...
+
+### Separate client and server X.509 PEM certificates and PKCS #8 keys
+
+Name | Description
+:--- | :---
+`plugins.security.ssl.transport.server.pemkey_filepath` | Path to the server certificate's key file (PKCS \#8). Must be specified using a relative path under the `config` directory. Required.
+`plugins.security.ssl.transport.server.pemkey_password` | The server key password. Omit this setting if the key has no password. Optional.
+`plugins.security.ssl.transport.server.pemcert_filepath` | Path to the X.509 node server certificate chain (PEM format). Must be specified using a relative path under the `config` directory. Required.
+`plugins.security.ssl.transport.server.pemtrustedcas_filepath` | Path to the root CAs (PEM format). Must be specified using a relative path under the `config` directory. Required.
+`plugins.security.ssl.transport.client.pemkey_filepath` | Path to the client certificate's key file (PKCS \#8). Must be specified using a relative path under the `config` directory. Required.
```
Comment (natebower) [inline _security/configuration/tls.md:92]:
```suggestion
`plugins.security.ssl.transport.client.pemkey_filepath` | The path to the client certificate's key file (PKCS \#8). Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 28]
```diff
@@ -54,27 +54,61 @@ Name | Description
...
+`plugins.security.ssl.transport.client.pemkey_password` | The client key password. Omit this setting if the key has no password. Optional.
+`plugins.security.ssl.transport.client.pemcert_filepath` | Path to the X.509 node client certificate chain (PEM format). Must be specified using a relative path under the `config` directory. Required.
```
Comment (natebower) [inline _security/configuration/tls.md:94]:
```suggestion
`plugins.security.ssl.transport.client.pemcert_filepath` | The path to the X.509 node client certificate chain (PEM format). Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 29]
```diff
@@ -54,27 +54,61 @@ Name | Description
...
+`plugins.security.ssl.transport.client.pemtrustedcas_filepath` | Path to the root CAs (PEM format). Must be specified using a relative path under the `config` directory. Required.
```
Comment (natebower) [inline _security/configuration/tls.md:95]:
```suggestion
`plugins.security.ssl.transport.client.pemtrustedcas_filepath` | The path to the root CAs (PEM format). Must be specified using a relative path under the `config` directory. Required.
```

File: _security/configuration/tls.md [hunk 30]
```diff
@@ -54,27 +54,61 @@ Name | Description
...
+
+### Separate client and server keystore and truststore files
+
+Name | Description
+:--- | :---
+`plugins.security.ssl.transport.keystore_type` | The type of the keystore file, either `JKS` or `PKCS12/PFX`. Optional. Default is `JKS`.
+`plugins.security.ssl.transport.keystore_filepath` | Path to the keystore file. Must be specified using a relative path under the `config` directory. Required.
```
Comment (natebower) [inline _security/configuration/tls.md:102]:
```suggestion
`plugins.security.ssl.transport.keystore_filepath` | The path to the keystore file. Must be specified using a relative path under the `config` directory. Required.
```

**Review by natebower — COMMENTED** [review]
@AntonEliatra @Naarcha-AWS Please see my changes and let me know if you have any questions. Thanks!
